assets/base.css

```diff
@@ -598,6 +598,10 @@ h6:empty {
   margin-left: 1rem;
 }
 
+.link:not([href]) {
+  cursor: not-allowed;
```

Comment:
I think there's still value in having it (`a.link`), but it definitely needs more specificity, since the `.link` class is used on non anchor tag elements.

A merchant could include a label and no link, and in this case we would be showing it. So I think it's still important to have a visual cue that tells a customer/merchant that the link does not go anywhere.